src/Employee.java

```diff
@@ -0,0 +1,109 @@
+import java.util.Objects;
+
+public class Employee {
+
+    //    Имя сотрудника
+    private String name;
+    //    Фамилия сотрудника
+    private String middleName;
+    //    Отчество сотрудника
+    private String lastName;
+    //    Отдел
+    private int department;
+    //    Зарплата сотрудника
+    private int salary;
+    //    id
+    private int id = 0;
+
+
+    //    Счетчик для id
+    private static int idValue = 0;
+
+    //    Метод для увеличения счетчика Id
+    private void increaseidValue() {
+        id = idValue;
+        idValue += 1;
```

Comment:
```suggestion
        id = idValue++;
```
можно было так в одну строку, тогда метод не понадобится отдельный)